src/services/oidc-data.service.ts

```diff
@@ -24,7 +24,7 @@ export class OidcDataService {
         );
 
         return this.httpClient.get<T>(url, {
```

Comment:
comma too many?

Reply:
no because of git code styling --> use trailing commas